src/Provider/Provider.Web/Orchestrators/DashboardOrchestrator.cs

```diff
@@ -88,54 +121,36 @@ private async Task<List<VacancySummary>> GetVacanciesAsync(long ukprn)
 
         private int SanitizePage(int page, int totalVacancies)
         {
-            return (page < 0 || page > (int)Math.Ceiling((double)totalVacancies / VacanciesPerPage)) ? 1 : page;
+            return page < 0 || page > (int)Math.Ceiling((double)totalVacancies / VacanciesPerPage) ? 1 : page;
         }
 
-        private VacancyStatus? SanitizeFilter(string filter)
+        private FilteringOptions? SanitizeFilter(string filter)
```

Comment:
I don't think this function should return a nullable now we have `FilteringOptions`
Instead of returning null, shouldn't this return `FilteringOptions.All`.